libbpf-rs/src/program.rs

```diff
@@ -110,6 +111,15 @@ pub enum ProgramAttachType {
     TraceFexit,
     ModifyReturn,
     LsmMac,
+    TraceIter,
+    CgroupInet4Getpeername,
+    CgroupInet6Getpeername,
+    CgroupInet4Getsockname,
+    CgroupInet6Getsockname,
+    XdpDevmap,
+    CgroupInetSockRelease,
+    XdpCpumap,
+    SkLookup,
```

Comment:
while you are at it, why not add the rest of values:

```
 994 ▸       BPF_SK_SKB_VERDICT,¬
 995 ▸       BPF_SK_REUSEPORT_SELECT,¬
 996 ▸       BPF_SK_REUSEPORT_SELECT_OR_MIGRATE,¬
 997 ▸       BPF_PERF_EVENT,¬
```

Reply:
Sounds good. Done!